src/nodes/NodeManager.ts

```diff
@@ -1190,6 +1190,10 @@ class NodeManager {
         (address) => `${address.host}:${address.port}`,
       )}`,
     );
+    if (seedNodes.length === 0) {
+      logger.debug(`No seed nodes provided, skipping discovery`);
+      return;
+    }
```

Comment:
What is this about? Why is this here?

Reply:
Technically out of scope of the PR. I added it to remove logger spam during tests. It is technically a fix that is needed.

Reply:
Which issue is this for? I thought we had discussed earlier and it already been fixed.

Reply:
If this is in fact a proper fix, please create an issue and fix it with this PR, and give demonstration of what exactly this fixes. Like if it is to reduce logger spam. Then post an example of what is expected or not expected during execution by copying the terminal output.

Reply:
Ok, I'll do that.